client/src/legacy/LeftAndMain.Tree.js

```diff
@@ -142,9 +142,11 @@ $.entwine('ss.tree', function($){
               }
             }
           })
-          .bind('move_node.jstree', moveNodeCallback)
+          .on('move_node.jstree', moveNodeCallback)
           // Make some jstree events delegatable
-          .bind('select_node.jstree check_node.jstree uncheck_node.jstree', function(e, data) {
+          .on('select_node.jstree check_node.jstree uncheck_node.jstree', function(e, data) {
+            // Clear out namespace before triggering so entwine handlers are called
+            e.namespace = '';
```

Comment:
This didn't need to be set to blank in the past because of a bug in jQuery where the namespace wasn't actually kept on the event after `triggerHandler()` was called. This bug has been fixed in the current version of jQuery, so we have to manually remove the namespace for it to get to entwine.